.github/workflows/pull-request.yml

```
@@ -0,0 +1,500 @@
+          for file in ${{ steps.changed-files.outputs.all_files }}; do
+            if [[ "$file" == *.ts || "$file" == *.tsx ]]; then
+              npx tsc --noEmit "$file"
+            fi
+          done
+
+      - name: Present and Upload coverage to Codecov as ${{env.CODECOV_UNIQUE_NAME}}
+        uses: codecov/codecov-action@v4
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          verbose: true
+          gcov_ignore: 'docs/'
```

actionlint (1.7.4)

input "gcov_ignore" is not defined in action "codecov/codecov-action@v4". available inputs are "codecov_yml_path", "commit_parent", "directory", "disable_file_fixes", "disable_safe_directory", "disable_search", "dry_run", "env_vars", "exclude", "fail_ci_if_error", "file", "files", "flags", "git_service", "handle_no_reports_found", "job_code", "name", "network_filter", "network_prefix", "os", "override_branch", "override_build", "override_build_url", "override_commit", "override_pr", "plugin", "plugins", "report_code", "root_dir", "slug", "token", "url", "use_legacy_upload_endpoint", "use_oidc", "verbose", "version", "working-directory"

(action)